FirstProject/new-evaluation.html

```diff
@@ -41,23 +39,23 @@
             <h2 class="description_section" style="color: #03A9F4;">Technical level</h2>
         </div>
 
-        <div class="row" style="background-color: #03A9F4; margin: auto; width: 73.1%; display: flex;">
+        <div class="row" style="background-color: #03A9F4; margin: auto; width: 73.2%; display: flex;">
```

Comment:
Of course that I had the posibility to choose an other width value. I chose that in a rondom way. I wanted all that divs to be equal as width.

When you resize the browser the value of the width will remain the same, but will be 73,1% from the smaller or bigger window.

If you resize it to 1200px(for example) the width will be 73,1% from 1200px.

If you resize it to 768px(for example) the width will be 73,1% from 768px.